src/Security/Member.php

```diff
@@ -914,16 +914,18 @@ public function onBeforeWrite()
             && static::config()->get('notify_password_change')
             && $this->isInDB()
         ) {
-            Email::create()
+            $email = Email::create()
                 ->setHTMLTemplate('SilverStripe\\Control\\Email\\ChangePasswordEmail')
                 ->setData($this)
                 ->setTo($this->Email)
                 ->setSubject(_t(
                     __CLASS__ . '.SUBJECTPASSWORDCHANGED',
                     "Your password has been changed",
                     'Email subject'
-                ))
-                ->send();
+                ));
+
+            $this->extend('updateChangedPasswordEmail', $email);
```

Comment:
Should we pass the member to the extension point here similarly to the lost password one, for consistency?
```suggestion
            $this->extend('updateChangedPasswordEmail', $email, $this);
```

Reply:
This one has the member already, by virtue of `extend()` being called on the member object.
I think if we want to be more consistent it would be better to change the other call to also be from member (so one extension class applied to the `Member` class can implement both extension methods) rather than changing this one.

Reply:
Of course, through the `->getOwner()` call... 🙈 

Yeah, we could do `$member->extend('updateForgotPasswordEmail', $email);`

Reply:
@GuySartorelli good idea! I’ve updated the forgot password call so the hook is now called on the `$member` object